selfdrive/controls/controlsd.py

```diff
@@ -80,7 +82,7 @@ def data_sample(CI, CC, sm, can_sock, state, mismatch_counter, can_error_counter
 
   events = list(CS.events)
   events += list(sm['dMonitoringState'].events)
-  add_lane_change_event(events, sm['pathPlan'])
+  add_lane_change_event(events, sm['pathPlan'], CS.leftBlindspot or CS.rightBlindspot)
```

Comment:
This would also need the direction check. You only want the alert to show if it's in the direction of the lane change.

Reply:
Done